vue/sbc-common-components/src/plugins/vuetify.ts

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
```

Comment:
do we need vuetify? Doesnt it work without vuetify?
may be try removing it and see if it works.